pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -657,43 +663,36 @@ def _create_default_config(
             cfg = {"train_micro_batch_size_per_gpu": logging_batch_size_per_gpu, **cfg}
         return cfg
 
-    def _filepath_to_dir(self, filepath: str) -> str:
-        return os.path.dirname(filepath)
-
     @property
     def deepspeed_engine(self):
         return self.model
 
+    @property
+    def _multi_device(self) -> bool:
```

Comment:
Should this be `multi_node` instead?

Reply:
it checks both, so maybe it just should be `is_distributed`?

Reply:
Yup, as we already have a property for that in `TrainingTypePlugin`

Reply:
ahh didn't know that, hesitant to overwrite the function as it's used to select distributed kwargs and stuff I think. Maybe its best for now to keep it as is?

Reply:
Sure, works. But would like to have a comment for future reference.